servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1141,6 +1148,40 @@ private void propagatePersisted(Inode inode, boolean replayed)
   }
 
   /**
+   * Create the permissionStatus for file or directory.
+   * @param remote true if the request is for creating permission from client side, the
+   * username binding into inode will be gotten from {@code AuthorizedClientUser.get().getName()}.
+   * If the remote is false, the username binding into inode will be gotten from {@link LoginUser}
+   * @return permissionStatus
+   * @throws IOException
+   */
+  private PermissionStatus createPermissionStatus(boolean remote) {
```

Comment:
Nice catch! Move this method to `PermissionStatus` and make it static. `FileSystemMaster` could invoke `PermissionStatus.get(TachyonConf conf, boolean remote)` to get a permission status.